opentech/static_src/src/sass/apply/components/_status-bar.scss

```diff
@@ -0,0 +1,178 @@
+.status-bar {
+    $root: &;
+    display: none;
+    padding: 30px 10px 80px;
+
+    @include media-query(tablet-portrait) {
+        display: flex;
+    }
+
+    &--mobile {
+        @include media-query(tablet-portrait) {
+            display: none;
+        }
+    }
+
+    &__subheading {
+        display: inline-block;
+        padding: 5px 10px;
+        margin: 10px 0 0;
+        background: $color--tomato;
+    }
+
+    &__icon {
+        position: absolute;
+        top: -10px;
+        left: 0;
+        z-index: 30;
+        display: none;
+        width: 20px;
+        height: 20px;
+
+        .status-bar__item--is-complete & {
+            display: block;
+        }
+
+        .status-bar__item:first-of-type & {
+            left: -10px;
+        }
+    }
+
+    &__item {
+        position: relative;
+        flex: 1;
+        height: 3px;
+        background: $color--mid-grey;
+
+        // every items dot
+        &::before {
+            position: absolute;
+            top: -10px;
+            left: 0;
+            width: 20px;
+            height: 20px;
+            background: $color--dark-grey;
+            border: 5px solid $color--mid-grey;
+            border-radius: 50%;
+            content: '';
+        }
+
+        // last items dont have a dot
```

Comment:
Is this comment is opposite of the case? Should it be last item displays dot only?